### ISSUE TITLE
There is an install error at 4.2.2 version

### ISSUE THREAD

**Issue by temacurbflow** [Cogoport/cogo-toast#53]
```
error /Users/user/path/node_modules/cogo-toast: Command failed.
Exit code: 1
Command: cd docs && yarn install
Arguments: 
Directory: /Users/user/path/node_modules/cogo-toast
Output:
/bin/sh: line 0: cd: docs: No such file or directory
```

macos

**Comment by harshzalavadiya** [comment]
same reproducable on linux as well

**Comment by avindra** [comment]
Same. Calling that command `install` registers it as part of the npm installation lifecycle:

https://docs.npmjs.com/misc/scripts

You can probably fix this by just calling it something else.

**Comment by avindra** [comment]
@anmolmahatpurkar  maybe you can call it as `install:doc` or something like that? not totally clear on the intent of that script.

**Comment by anmolmahatpurkar** [comment]
This was an unintended mistake. My apologies. Fixed in v4.2.3 :)
